src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -166,6 +166,14 @@ internal View(ViewImpl implementation, bool shown = true) : this(Interop.View.Ne
             }
         }
 
+        public struct WantToUseViewImpl
```

Comment:
As this struct is open public, I think we need to name it more carefully...
WantToUse is too informal and ViewImpl is not open public so I'm not sure using this name in public struct is good enough.